pod/video/forms.py

```diff
@@ -532,10 +532,6 @@ def set_queryset(self):
             self.fields["channel"].queryset = \
                 self.fields["channel"].queryset.filter(
                     sites=Site.objects.get_current())
-        if "theme" in self.fields:
```

Comment:
Nope il faut le laisser ! ou tout du moins le renseigner à partir des chaines (self.themes = channel in self.channel)

Reply:
Je l'ai remis et je l'ai modifié